tests/logger.js

```diff
@@ -53,6 +53,22 @@ exports.log = {
         test.ok(this.logger.log('INFO', 'another test info'));
         test.done();
     },
+    'log in json' (test) {
+        this.logger.deferred_logs = [];
+        test.expect(3);
+        this.logger.format = this.logger.formats.JSON;
+        test.equal(0, this.logger.deferred_logs.length);
+        test.ok(this.logger.log('WARN','test warning'));
+        test.equal(1, this.logger.deferred_logs.length);
+        test.done();
+    },
+    'log in json w/deffered' (test) {
```

Comment:
```suggestion
    'log in json w/deferred' (test) {
```

Reply:
Changed this and the other "deffered" on the logfmt test too